frontend/src/components/MessageLoaddingSkelton.jsx

```
@@ -0,0 +1,15 @@
+import React from 'react'
+
+const MessageLoaddingSkelton = () => {
+  return (
+    <div className='max-w-3xl mx-auto space-y-6'>
+           {[...Array(6)].map((_,index)=>(
+            <div className={`${index%2===0?"chat-start":"chat-end"} animate-pulse`}>
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)